README.md

```diff
@@ -14,27 +14,47 @@ Ailly's best feature is rapid prompt engineering iteration. By keeping your prom
 
 To get started, follow these steps:
 
-1. Create a folder named `content`.
-2. Create a file named `content/.aillyrc`, and put your top-level prompt instructions in the file.
-   - Include system prompts, level setting expectations. etc.
-3. Create more numbered files, such as `content/01_big_point.md` and `content/02_second_point.md`.
-4. Run Ailly using NodeJS: `npx @ailly/cli --root content`
+1. Create a folder named `jokes`.
+2. Create a file named `10_chickens.md` with "Tell me a joke about chickens" as the content.
+3. Run Ailly using NodeJS: `npx @ailly/cli --root content`
+   - See the joke in `10_chickens.md.ailly.md`
+2. Create a file named `content/.aillyrc` with "You are a farmer writing jokes for your other barnyard animals."
```

Comment:
Any greymatter here or just the plaintext?

Reply:
Just plaintext at this point in the Quickstart.